controllers/operands/networkAddons_test.go

```diff
@@ -540,9 +540,11 @@ var _ = Describe("CNA Operand", func() {
 				existingCNAO.Spec.KubeSecondaryDNS = &networkaddonsshared.KubeSecondaryDNS{}
 			}
 
+			kubeSecondaryDNSNameServerIP := "127.0.0.1"
 			if o.setFeatureGate {
 				deployKubeSecondaryDNS := o.featureGateValue
 				hco.Spec.FeatureGates.DeployKubeSecondaryDNS = &deployKubeSecondaryDNS
+				hco.Spec.KubeSecondaryDNSNameServerIP = &kubeSecondaryDNSNameServerIP
```

Comment:
nit: you can use `pointer.String()` for that, e.g.
```suggestion
			if o.setFeatureGate {
				deployKubeSecondaryDNS := o.featureGateValue
				hco.Spec.FeatureGates.DeployKubeSecondaryDNS = &deployKubeSecondaryDNS
				hco.Spec.KubeSecondaryDNSNameServerIP = pointer.String("127.0.0.1")
```

Reply:
but i also use the value on the EXPECT